lib/registerTypes/RegisterTypesClientPacks.h

```diff
@@ -112,7 +112,6 @@ void registerTypesClientPacks(Serializer &s)
 	s.template registerType<CPackForClient, CArtifactOperationPack>();
 	s.template registerType<CArtifactOperationPack, PutArtifact>();
 	s.template registerType<CArtifactOperationPack, EraseArtifact>();
-	s.template registerType<CArtifactOperationPack, MoveArtifact>();
```

Comment:
This might break saves. Can you check whether saves made before this PR can still be loaded in this PR?

Or just add dummy pack with comment / static_assert, e.g. 
```cpp
static_assert(ESerializationVersion::RELEASE_150 < ESerializationVersion::CURRENT, "Please remove this compatibility check once it no longer needed");
```

Reply:
> Can you check whether saves made before this PR can still be loaded in this PR?

Yep Can be loaded.
As I see MoveArtifact was used only for network packets